stock_move_change_source_location/wizards/stock_move_change_source_location_wizard.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- wizard view -->
+    <record id="stock_move_change_source_location_view" model="ir.ui.view">
+        <field name="name">Stock Move Change Source Location</field>
+        <field name="model">stock.move.change.source.location.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Stock Picking">
+                <div>
+                    Change the source location.
+                    <br />
+                </div>
+                <group>
+                    <field name="warehouse_view_location_id" invisible="1" />
+                    <field name="moves_to_change" />
+                    <field
+                        name="old_location_id"
+                        attrs="{'invisible': [('moves_to_change', '!=', 'matched')]}"
+                        options="{'no_create': True, 'no_open': True}"
+                    />
+                    <field
+                        name="new_location_id"
+                        domain="[('id', 'child_of', warehouse_view_location_id)]"
```

Comment:
I think that due to this domain the view location is being added as an available location:

![image](https://user-images.githubusercontent.com/71635103/127104165-835d4f24-1b26-477e-8947-1c94e9b3a32c.png)

Reply:
I've added `("usage", "=", "internal")` in the domain and it should not appear anymore.

Thanks!